src/settings/SettingsCard.js

```diff
@@ -14,20 +14,31 @@ export default class SettingsCard extends React.Component {
     static propTypes = {
 		actions: React.PropTypes.object.isRequired,
         settings: React.PropTypes.object.isRequired,
+		loginid: React.PropTypes.string.isRequired,
     };
 
 	render() {
-		const {settings, actions} = this.props;
+		const {loginid} = this.props;
+		const isVirtual = (loginid.startsWith('VRTC'));
```

Comment:
extra parentheses * 
and I did something stupid elsewhere, and this is copied pasted from the stupid code, will fix